src/olympia/abuse/tasks.py

```diff
@@ -29,6 +29,14 @@
 log = olympia.core.logger.getLogger('z.abuse')
 
 
+task_retry_kw = {
+    'autoretry_for': (requests.RequestException,),
+    'retry_backoff': 30,  # start backoff at 30 seconds
+    'retry_backoff_max': 60 * 60,  # Max out at 1 hour between retries
+    'retry_kwargs': {'max_retries': 79},  # this works out about 72 hours
```

Comment:
Whether `max_retries` should be directly defined in the task kw, or inside `retry_kwargs` was ambiguous - and I'm not sure we do it consistently in our code either.  I went with this way because of the 2nd example in https://docs.celeryq.dev/en/latest/userguide/tasks.html#automatic-retry-for-known-exceptions